Makefile

```diff
@@ -116,6 +116,10 @@ check: force-yarn generate node_modules
 
 check-all: check test smoketest cy-wide-prod-run cy-mobile-prod-run
 
+EXPLORE_BUILD = $(shell find graphql2/explore/build)
```

Comment:
This is only going to find files that exist, meaning it won't register targets until AFTER being built, since we know them we can add them directly.